src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+public abstract class Task {
+
+    protected String description;
+    protected boolean taskIsDone;
+    protected int taskIsDoneState;
+
+    public Task(String description) {
+        this.description = description;
+        this.taskIsDone = false;
+        this.taskIsDoneState = 0;
```

Comment:
You can create a public method String getTaskState() { return taskIsDone ? "0" : "1";} To avoid redundant variable. And the this. at line 11 and line 12 is violating the coding standard I believe. The this. should be removed.